### PR TITLE
Reworked config page and dashboard removed ? from overview widget, made small s…

### PR DESCRIPTION
**Pull request by EmileEttel**
Reworked config page and  removed ? from overview widget, made small slivers not show percentages on graph widget

### CODE REVIEW

**Comment by STom6** [comment]
- [X] Greffe page config sur main
- [X] Refaire les changements rapides du dashboard (retirer les "?" qui cache l'affichage)
- [X] Retirer les 0%

File: src/assets/translation/fr.json
Comment (STom6) [inline src/assets/translation/fr.json:1]: maybe do the same for the description

File: src/components/Settings/IntegrationWidget.vue [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+    >
+      Configurer
+      l'intégration Google Drive
+    </v-tab>
 
     <!-- Button to configure Notion integration -->
-    <v-btn block class="rounded-rectangle" @click="showDialog = true">Configurer l'intégration Notion</v-btn>
+    <v-tab block class="integrate-button" @click="showDialog = true">Configurer l'intégration Notion</v-tab>
 
     <!-- Dialog popup for Notion integration -->
```
Comment (STom6) [inline src/components/Settings/IntegrationWidget.vue:14]: remove gpt comments

File: src/components/dashboard/widgets/OverView/FileItem.vue [hunk 2]
```diff
@@ -136,14 +119,18 @@ export default {
...
-      return data;
+      return([]);
+      // let data = [];
+      // const configurations = this.getConfigurations(tidyScore, tab);
+      // if (configurations != null && configurations.length > 0) {
+      //   for (let i = 0; i < configurations.length; i++) {
+      //     data.push(configurations[i].grade);
+      //   }
+      // }
+      // return data;
```
Comment (STom6) [inline src/components/dashboard/widgets/OverView/FileItem.vue:133]: remove comments

**Comment by STom6** [comment]
From my point of view, the PR can be merged, but there are modifications to be made in it or in a new issue.

**Review by Bootoyka — APPROVED** [review]
lgtm, merging

**Pull request merged**